### ISSUE TITLE
live_tester in training problem

### ISSUE THREAD

**Issue by weiyichenlun** [swook/GazeML#25]
https://github.com/swook/GazeML/blob/7087b9eb49a5749d84016a2079a91e85d7984511/src/core/model.py#L90
Hi, the work is really good. When running the DPG code, I encountered some problems. As you can see in the link, the model initialized live_tester during model initialization, however, the use_batch_statistics_at_test was set to True, which lead to  during testing in do_full_test function, the use_batch_statistics was set to True. So, my question is , why set the is_training in batch_norm function to True during test? I have tried to set it to false and during the training, I found the test result was very bad.

**Comment by swook** [comment]
Yes, for the model to work you must use mini-batch statistics at test time. I believe this is because the distribution of pixel values are greatly different but only in terms of scale and bias.

This is not a hack per se, as you will find that many deep learning papers use mini-batch stats. at test time in contrast to the original batch norm paper's suggestions.
